ClientApp/src/components/shared/EmailContentBox.tsx

```diff
@@ -0,0 +1,110 @@
+import {DefaultEditor} from "react-simple-wysiwyg"
+import React from "react";
+import { Button, Dialog, Grid, IconButton, Input, TableBody, TableCell, TableRow, TextField, Typography } from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+import { GiverType } from "./Types";
+import ApiService from "common/functions/apiServiceClass";
+import { useAuth0 } from "@auth0/auth0-react";
+import { FC, useEffect, useState } from "react";
+import SendIcon from '@material-ui/icons/Send';
+
+
+interface ISendSingleEmail {
+
+    open: boolean;
+    handleClose: () => void;
+    giver: GiverType;
+ }
+
+
+const SendSingleEmail: React.FC<ISendSingleEmail> = (
```

Comment:
Consider name change, either on the file itself or the component. The component and file should have the same/similar name

Reply:
Fixed